api/jobset/v1alpha2/jobset_types.go

```diff
@@ -198,6 +198,12 @@ type Network struct {
 	// Defaults to <jobSet.name> if not set.
 	// +optional
 	Subdomain string `json:"subdomain,omitempty"`
+
+	// PublishNotReadyAddresses indicates that any agent
+	// which deals with endpoints for this Service should disregard any indications of ready/not-ready.
+	// Defaults to true if not set
```

Comment:
"Indicates if DNS records of pods should be published before the pods are ready. Defaults to True."

Reply:
updated.